core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -157,49 +164,63 @@ private void parseArguments(final String[] args) throws IOException {
         }
     }
 
-    private void resetInputAndInternalAndSeekToEndIntermediateTopicOffsets() {
+    private void mayberesetInputAndInternalAndSeekToEndIntermediateTopicOffsets() {
         final List<String> inputTopics = options.valuesOf(inputTopicsOption);
         final List<String> intermediateTopics = options.valuesOf(intermediateTopicsOption);
 
-        if (inputTopics.size() == 0 && intermediateTopics.size() == 0) {
-            System.out.println("No input or intermediate topics specified. Skipping seek.");
-            return;
-        } else {
-            if (inputTopics.size() != 0) {
-                System.out.println("Resetting offsets to zero for input topics " + inputTopics + " and all internal topics.");
-            }
-            if (intermediateTopics.size() != 0) {
-                System.out.println("Seek-to-end for intermediate topics " + intermediateTopics);
+        final List<String> internalTopics = new ArrayList<>();
+
+        final List<String> notFoundInputTopics = new ArrayList<>();
+        final List<String> notFoundIntermediateTopics = new ArrayList<>();
+
+        Boolean dryRun = options.has(dryRunOption);
+        String groupId = options.valueOf(applicationIdOption);
+
+        if (!dryRun) {
```

Comment:
Added the check to make output clear.
As with out checking input topics, existed or not, we print these input topics offset will be seek to end or beginning.
In dry-run i have taken care that only for topics, which exist print seek to end/begininng. for remaining topics print no action on them.